docs/encoder.md

```diff
@@ -42,27 +43,49 @@ SDA = board.GP0
 SCL = board.GP1
 i2c = busio.I2C(SCL, SDA)
 
-encoder_handler.pins = ((i2c, 0x36, False),)
+encoder_handler.pins = ((i2c, 0x36, False), (encoder 2 definition), etc. )
 ```
 
+3. Define the mapping of keys to be called for each layer.
 
-3. Define the mapping of keys to be called (1 / layer)
 ```python
 # You can optionally predefine combo keys as for your layout
 Zoom_in = KC.LCTRL(KC.EQUAL)
 Zoom_out = KC.LCTRL(KC.MINUS)
 
-
-encoder_handler.map = [(( KC.VOLD, KC.VOLU, KC.MUTE),(encoder 2 definition), etc. ), # Layer 1
-                      ((KC.Zoom_out, KC.Zoom_in, KC.NO),(encoder 2 definition), etc. ), # Layer 2
-                      ((KC.A, KC.Z, KC.N1),(encoder 2 definition), etc. ), # Layer 3
-                      ((KC.NO, KC.NO, KC.NO),(encoder 2 definition), etc. ), # Layer 4
+encoder_handler.map = [(( KC.VOLD, KC.VOLU, KC.MUTE), (encoder 2 definition), etc. ), # Layer 1
+                      ((Zoom_out, Zoom_in, KC.NO), (encoder 2 definition), etc. ), # Layer 2
+                      ((KC.A, KC.Z, KC.N1), (encoder 2 definition), etc. ), # Layer 3
+                      ((KC.NO, KC.NO, KC.NO), (encoder 2 definition), etc. ), # Layer 4
                       ]
 ```
 
+## Encoder resolution
+
+Depending on your encoder precision, it may send 4 or 2 pulses on every detent. By default the encoder resolution is set to 4, but if your an encoder only activates on every second detent (skips pulses), set the resolution to 2. If the encoder activates twice on every detent, set the value to 4.
+
+You can change the default globally for all encoders **before** initializing the encoder pins (`main.py` file):
+```python
+encoder_handler.resolution = 2
+encoder_handler.pins = ( (board.GP14, board.GP15, None), (board.GP26, board.GP27, None), )
+```
+
+Or if you have different types of encoders, set resolution for each encoder individually:
+```python
+encoder_handler.pins = (
+    (board.GP14, board.GP15, None, False, 4), 
+    (board.GP26, board.GP27, None, False, 2),
+    (board.GP26, board.GP27, None ), # will be set to global default
+)
+```
+
+## Handler methods overrides
 
+Encoder methods `on_move_do` and `on_button_do` can be overridden for complex use cases.
+
```

Comment:
```suggestion
```

Reply:
This was in the original docs so I thought I'd keep it. I think it can be useful to know it's possible even though no examples are given at the moment.

Reply:
Ah, sorry, for some reason I thought this meant removing text. Thanks for your edits! :)